src/components/ActionsMenu/ActionsMenu.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import styles from './ActionsMenu.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { ActionsMenuItem } from './Item/ActionsMenuItem';
+import { ACTIONS_MENU__DATA_KEYS } from './dataHooks';
+import { Divider } from '../Divider';
+
+export enum Alignment {
+  left = 'left',
+  center = 'center',
+  right = 'right',
+}
+
+export interface ActionsMenuProps {
+  alignment?: Alignment;
+}
+
+interface DefaultProps {
+  alignment: Alignment;
+}
+
+function getDataAttributes(mobile: boolean) {
```

Comment:
I think it's better to have this function as a class method, no?

Reply:
now it has no deps from props or class instance, so I've put it aside.
Could do it as static method though

Reply:
moved to class